runtime/tools/spec-builder/index.js

```diff
@@ -1,8 +1,28 @@
-const fs = require('fs');
+#!/usr/bin/env node
 const GenesisBuilder = require('./src/genesis-builder');
-const customSpec = require('./customSpec.json');
+const fs = require('fs');
+const curPath = process.cwd();
+
+let argv = require('yargs')
+    .usage('Usage: $0 <command> [options]')
+    .command('build-spec', 'Converts custom spec json file to raw')
+    .example('$0 build-spec -f customSpec.json -o customSpecRaw.json', 'convert given file to raw and write to output')
+    .alias({'f' : 'file', 'o' : 'output'})
+    .nargs({'f' : 1, 'o': 1})
+    .describe('f', 'Load a file')
+    .demandOption(['f'])
+    .help('h')
+    .alias('h', 'help')
+    .epilog('copyright 2020')
+    .argv;
+
+let customSpec = require(`${curPath}/${argv.file}`);
```

Comment:
What if there is no such file? I think that we need to handle that error and return a response that no such file exists